creme/settings.py

```diff
@@ -951,7 +951,8 @@
 CREME_OPTLIB_JS = [
     ('creme.activities', 'activities/js/jquery/extensions/fullcalendar-3.10.2.js'),
     ('creme.geolocation', 'geolocation/js/lib/leaflet-1.6.0.js'),
-    ('creme.sketch', 'sketch/js/lib/d3-5.16.0.js'),
+    # ('creme.sketch', 'sketch/js/lib/d3-5.16.0.js'),
```

Comment:
I presume you can remove this line, because it was not in creme 2.3 and you removed the JS file anyway.